conda_env/specs/jinja2_file.py

```diff
@@ -0,0 +1,36 @@
+try:
+    import jinja2
+except ImportError:
+    jinja2 = None
+from .. import env
+from ..exceptions import EnvironmentFileNotFound, Jinja2NotInstalled
+
+
+class Jinja2Spec(object):
+    _environment = None
+
+    def __init__(self, filename=None, **kwargs):
+        self.filename = filename
+        self.msg = None
+
+    def can_handle(self):
```

Comment:
Calling `can_handle` multiple times will load the file multiple times, I don't think it really matters, but just letting you know :)